shared-libs/transitions/src/transitions/mark_for_outbound.js

```diff
@@ -0,0 +1,60 @@
+const vm = require('vm');
+
+const config = require('../config'),
+      db = require('../db'),
+      logger = require('../lib/logger');
+
+const CONFIGURED_PUSHES = 'outbound';
+
+const arrayinate = object => Object.keys(object).map(k => {
```

Comment:
Not exactly. It's a terrible name, I should have made a TODO to change it but I forgot.

It's converting:

```
{
  foo: {
    some: 'config'
  },
  bar: {
    different: 'config altogether'
  }
}
```

into:

```
[
  {key: 'foo', some: 'config'},
  {key: 'bar', different': 'config altogether'}
]
```

Basically you want to store thing as keys and values in config because it reads cleaner, but you then want to those two pieces of information combined when you process.

I could use `Object.entries` instead and pass around a list of tuples?

Reply:
> Basically you want to store thing as keys and values in config because it reads cleaner

Is this something we do in other areas of config already? Or a new pattern.

Reply:
I dropped the use of it in both modules and refactored around it